src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -10,6 +11,7 @@ export default class VizShonenJump extends Connector {
         this.tags = [ 'manga', 'english' ];
         this.url = 'https://www.viz.com';
         this.requestOptions.headers.set( 'x-referer', this.url );
+        this.EntryTypeEnum = Object.freeze({"volume":1, "chapter":2});
```

Comment:
is there a reason for the `Object.freeze`, i don't see it, nor can i really test it

Reply:
I'm not sure, I just looked up how to do enums in JS and this was recommended. From the docs, it seems like this carries a level of immutability with it, so EntryTypeEnum can't be accidentally modified.: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze